plugins/base/src/main/kotlin/transformers/documentables/ModuleAndPackageDocumentationTransformer.kt

```diff
@@ -14,7 +14,7 @@ import java.nio.file.Paths
 
 internal class ModuleAndPackageDocumentationTransformer(val context: DokkaContext) : PreMergeDocumentableTransformer {
 
-    override fun invoke(original: List<DModule>): List<DModule> {
+    override fun invoke(modules: List<DModule>, context: DokkaContext): List<DModule> {
```

Comment:
Another context is redundant here, as it is passed through constructor

Reply:
See discussion above ☺️